week2/velog/app.js

```diff
@@ -1,5 +1,22 @@
 const weekSelectButton = document.querySelector('.week-select-button');
 const dropContent = document.querySelector(".drop-content");
+const weekOption = document.querySelectorAll('.week-option');
+const selectedWeek = document.querySelector('.select-week');
+
+const selectedWeekOption = (e) => {
+    e.preventDefault();
+    e.target.classList.add("checked")
+    selectedWeek.innerHTML = e.target.innerHTML;
+    dropContent.classList.remove("toggle-week-options");
+    for( let i=0; i<weekOption.length; i++){
+        if(weekOption[i].innerHTML != e.target.innerHTML)
+            weekOption[i].classList.remove("checked");
+    }
+}
+
+for( let i=0; i<weekOption.length; i++){
+	weekOption[i].addEventListener("click", selectedWeekOption);
+}
```

Comment:
```suggestion
weekOption.forEach((option)=>{
       option.addEventListener("click", selectedWeekOption);
});
```
forEach도 익혀보면 조을것가타! 여기서 사실 idx를 사용하지 않고있어서!!

Reply:
아 forEach를 사용해서도 할 수 있겠네! 
[for vs forEach vs map](https://fromnowwon.tistory.com/entry/forEach-map)